RadixWallet/Clients/SubmitTransactionClient/SubmitTransactionClient+Live.swift

```diff
@@ -10,7 +12,7 @@ extension SubmitTransactionClient: DependencyKey {
 
 			@Sendable func pollTransactionStatus() async throws -> GatewayAPI.TransactionStatusResponse {
 				let txStatusRequest = GatewayAPI.TransactionStatusRequest(
-					intentHash: txID.asStr()
+					intentHash: txID.bech32EncodedTxId
```

Comment:
Or `formatted(.raw)` which is the samething - also same: `.description` and `toRawString()`